PWGHF/HFC/TableProducer/correlatorDplusHadrons.cxx

```diff
@@ -39,8 +39,8 @@ double getDeltaPhi(double phiD, double phiHadron)
 
 /// definition of variables for Dplus hadron pairs (in data-like, MC-reco and MC-kine tasks)
 const int npTBinsMassAndEfficiency = o2::analysis::hf_cuts_dplus_to_pi_k_pi::nBinsPt;
-const double efficiencyDmesonDefault[npTBinsMassAndEfficiency] = {};
-auto efficiencyDmeson_v = std::vector<double>{efficiencyDmesonDefault, efficiencyDmesonDefault + npTBinsMassAndEfficiency};
+const double efficiencyDmesonDefault[npTBinsMassAndEfficiency + 1] = {};
+auto efficiencyDmeson_v = std::vector<double>{efficiencyDmesonDefault, efficiencyDmesonDefault + (npTBinsMassAndEfficiency + 1)};
```

Comment:
Just define a vector, there is no need to have a c-array defined first (and please remove `_v` from the variable name).